### PR TITLE
Codemirror

### PR DESCRIPTION
**Pull request by amicis31**
- Moved lint.js, codemirror_linter.js, lint.css to multilang plugin for linter service to work
- Erased linter button from tools.html of multilang plugin
- Added lintingOptions to common.js
- Moved url from javascript (codemirror_lint.js) to python code (code_multiple_languages_problem.py) in multilang plugin

Note: Make pull request to wrong repository :laughing:

### CODE REVIEW

File: inginious/frontend/plugins/multilang/problems/code_multiple_languages_problem.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
 path_to_plugin = os.path.abspath(os.path.dirname(__file__))
 python_tutor_url = "http://127.0.0.1:8003/"
+linter_url = "http://localhost:4567/"
```
Comment (amicis31) [inline inginious/frontend/plugins/multilang/problems/code_multiple_languages_problem.py]: Not sure how to asks for the url and port variables from the configuration.yaml file. Just copy the style of python_tutor_url

Comment (amrondonp) [reply inginious/frontend/plugins/multilang/problems/code_multiple_languages_problem.py]: I'll fix both

**Comment by amrondonp** [comment]
Cool, It looks good to me

**Comment by amrondonp** [comment]
@amicis31 We'll need to discuss this change. I understand that the solution here was to modify task.js. However, as the inginious original team updates minified.js version we will always have conflicts with them. So we need to double check if there are other feasible solutions to this problem. I wouldn't want to have a conflict in a minified file that nobody will understand.

By the way, It works on our testing server

**Pull request merged**